<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveForm</class>
 <widget class="QWidget" name="SaveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="title_save_label"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="enter_button_save_my_list">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
